hazelcast/src/main/java/com/hazelcast/cache/impl/client/CacheEntryProcessorRequest.java

```diff
@@ -110,4 +111,24 @@ public void setCompletionId(Integer completionId) {
         this.completionId = completionId != null ? completionId : -1;
     }
 
+    @Override
+    public Permission getRequiredPermission() {
+        return null;
+    }
+
+    @Override
+    public Object[] getParameters() {
+        return new Object[]{key, null, arguments};
```

Comment:
Why second parameter is `null`? Because of `EntryProcessor` may not be `Serializable`?